tfutils/base.py

```diff
@@ -484,12 +497,18 @@ def save(self, train_res=None, valid_res=None, step=None, validation_only=False)
         rec['step'] = step
 
         if len(train_res) > 0:
-            # TODO: also include error rate of the train set to monitor overfitting
+            # TODO: also include error rate of the train set to monitor
+            # overfitting
             message = 'Step {} ({:.0f} ms) -- '.format(step, 1000 * duration)
-            msg2 = ['{}: {:.4f}'.format(k, v) for k, v in train_res.items() if k != 'optimizer' and k not in self.save_to_gfs]
+            msg2 = ['{}: {:.4f}'.format(k, v) for k, v in train_res.items()
+                    if k not in ['optimizer', 'grads', 'zero_grad'] and k not in self.save_to_gfs]
```

Comment:
@alexandonian actually maybe we should print out a summary of the gradients.   

btw, what is the "zero_grad" object?